src/adaptors/tcp_adaptor.c

```diff
@@ -1340,7 +1340,7 @@ void qd_dispatch_delete_tcp_connector(qd_dispatch_t *qd, void *impl)
     qd_tcp_connector_t *ct = (qd_tcp_connector_t*) impl;
     if (ct) {
         //need to close the pseudo-connection used for dispatching
-        //deliveries out to live connnections:
+        //deliveries out to live connections:
```

Comment:
I'd prefer to see changes like this typo fix pushed as a separate commit. It could be NO-JIRA even. Don't let it stop this PR though.

Reply:
allright; next time, then